solana/program_v2/packages/tests/src/gatekeeper-suite/set-gatekeeper-state.test.ts

```diff
@@ -91,7 +91,7 @@ describe('Gateway v2 Client', () => {
   });
 
   describe('Set Gatekeeper State', () => {
-    it("Should set a gatekeeper's state", async function () {
+    it.skip("Should set a gatekeeper's state", async function () {
```

Comment:
why is this skipped ?

Reply:
@JulianSpring skipped some of these during his testing... Unsure if he meant to leave them skipped?

Reply:
Oh sorry! I will fix this in my current task

Reply:
Not really sure why it is showing as a change though. It is part of feature/gateway_v2

Reply:
@JulianSpring @DiosAkhilleus it's fine, leave it then... we can fix it later